### PR TITLE
Add total usage job to pipeline and API

### PR DESCRIPTION
**Pull request by annapawlicka**
This PR:
- Adds total-usage to synthetic readings job in pipeline
- Adds `total-usage-weekly` and `total-usage-monthly` to repl methods
- Updates API to allow for creation of total usage datasets

### CODE REVIEW

File: src/clj/kixi/hecuba/api/datasets.clj [hunk 1]
```diff
@@ -67,7 +67,9 @@
       :subtract parent-period
       :divide "INSTANT"
       :multiply-series-by-field parent-period
-      :divide-series-by-field parent-period)))
+      :divide-series-by-field parent-period
+      :total-usage-weekly "PULSE"
+      :total-usage-monthly "PULSE")))
```
Comment (otfrom) [inline src/clj/kixi/hecuba/api/datasets.clj]: which one of these is the tariff calculation?

Comment (annapawlicka) [inline src/clj/kixi/hecuba/api/datasets.clj]: None - the card I'm currently working on is to add total usage calculation. The bottom two lines are adding those (weekly and monthly)

File: src/clj/kixi/hecuba/controller/repl.clj [hunk 2]
```diff
@@ -12,23 +12,28 @@
...
+(defreplmethods mislabelled-check :dest :data-quality :type :mislabelled-sensors)
+(defreplmethods median-spike-check :dest :data-quality :type :spike-check)
+(defreplmethods synthetic-readings :dest :calculated-datasets :type :synthetic-readings)
+(defreplmethods resolution :dest :data-quality :type :resolution)
+(defreplmethods convert-to-co2 :dest :calculated-datasets :type :convert-to-co2)
+(defreplmethods convert-to-kwh :dest :calculated-datasets :type :convert-to-kwh)
+(defreplmethods sensor-status-check :dest :data-quality :type :sensor-status)
+(defreplmethods actual-annual-calculation :dest :calculated-fields :type :actual-annual)
+(defreplmethods total-usage-weekly :dest :calculated-datasets :type :total-usage :operation :weekly)
+(defreplmethods total-usage-monthly :dest :calculated-datasets :type :total-usage :operation :weekly)
```
Comment (otfrom) [inline src/clj/kixi/hecuba/controller/repl.clj]: why did you get rid of the ->'s?

Comment (annapawlicka) [inline src/clj/kixi/hecuba/controller/repl.clj]: We discussed it with @sw1nn some time ago and `->` should be used when the job takes something from one place/format to another. I've initially blindly copied defreplmethods from another project and just replaced the names. This is correcting that mistake.

File: src/clj/kixi/hecuba/api/datasets.clj (hunk 1 above)
Comment (otfrom) [inline src/clj/kixi/hecuba/api/datasets.clj]: OK. There will be another PR for the tariff bits then I take it.

**Comment by otfrom** [comment]
:shipit:
